doxygen-awesome.css

```diff
@@ -1414,8 +1417,8 @@ div.memdoc, div.memproto, h2.memtitle {
 }
 
 div.memdoc {
-    padding: 0 var(--spacing-medium);
-    background: var(--page-background-color);
+    padding: 2px var(--spacing-medium);
```

Comment:
reason why i added `2px` to padding was, if i left it as it is, and then made the background color slightly lighter/darker than the main page's bg, it would result in smt like this :

![image](https://user-images.githubusercontent.com/90788311/206268308-61d02026-a57b-4b67-9e5f-76f7ec953680.png)

Reply:
I'd prefer this to avoid parent-child collapsing of the margins:
```suggestion
    overflow: auto;
```

Should have the same effect, but without the additional spacing.